packages/keyring-snap-bridge/src/SnapKeyring.ts

```diff
@@ -534,58 +554,126 @@ export class SnapKeyring extends EventEmitter {
     );
   }
 
+  /**
+   * Checks if a Snap ID is known from the keyring.
+   *
+   * @param snapId - Snap ID.
+   * @returns `true` if the Snap ID is known, `false` otherwise.
+   */
+  hasSnapId(snapId: SnapId): boolean {
```

Comment:
Do you think this method will be used by others or can we make it private?

Reply:
My idea was to expose it so that we would have a way from a the `SnapKeyring` to know which Snaps are being used.

But that's just me maybe 🙈 We can make it private for now if you prefer!